opm/simulators/flow/partitionCells.hpp

```diff
@@ -26,14 +26,28 @@
 
 namespace Opm
 {
-
-    /// Read a partitioning from file, assumed to contain one number per cell, its partition number.
-    /// \return pair containing a partition vector (partition number for each cell), and the number of partitions.
-    std::pair<std::vector<int>, int> partitionCellsFromFile(const std::string& filename, const int num_cells);
-
-    /// Trivially simple partitioner assigning partitions en bloc, consecutively by cell index.
-    /// \return pair containing a partition vector (partition number for each cell), and the number of partitions.
-    std::pair<std::vector<int>, int> partitionCellsSimple(const int num_cells, const int num_domains);
+class Well;
+
+template<class Grid>
```

Comment:
Please add a little documentation for this function, as well as for partitionCellsZoltan(), now that they will have wider exposure.

Reply:
done.